web/src/elements/utils/images.ts

```diff
@@ -1,7 +1,43 @@
-import { ResolvedUITheme } from "#common/theme";
+import { resolveUITheme, rootInterface } from "#common/theme";
 
-export function themeImage(rawPath: string, theme: ResolvedUITheme) {
-    return rawPath.replaceAll("%(theme)s", theme);
+import type { AKElement } from "#elements/Base";
+
+import { html, TemplateResult } from "lit";
+
+export const FontAwesomeProtocol = "fa://";
+
+export function themeImage(rawPath: string) {
+    const enabledTheme = rootInterface<AKElement>()?.activeTheme || resolveUITheme();
+
+    return rawPath.replaceAll("%(theme)s", enabledTheme);
+}
```

Comment:
Avoid including `rootInterface`. We've slowly deprecated its usage to avoid rendering issues when the browser triggers a theme change.

https://github.com/goauthentik/authentik/blob/46b889eab12cba1c37cab545c174b0b4dbcef762/web/src/elements/utils/images.ts#L3